service/routes.py

```diff
@@ -42,4 +44,19 @@ def index():
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+
 # Todo: Place your REST API code here ...
+@app.route("/orders/<int:orderid>", methods=["GET"])
+def get_order(orderid):
```

Comment:
Can we create a test for this too? I suppose we'll need to create tests in both test_models.py and test_routes.py. I think we won't be able to run them successfully though until we implement and merge  the CREATE ORDER API, but you can comment your added tests and then uncomment and test them once we put in the create order.

Reply:
You're right, I think so. This PR should not be merged now. I posted this PR before I watched the tutorial video XD. I will add the test cases soon.